app/controllers/schools/faculty_controller.rb

```diff
@@ -51,5 +58,12 @@ def faculty
           policy_class: Schools::FacultyPolicy
         )
     end
+
+    # work around for mismatch of archived_at key
+    def transformed_params(params)
+      params[:faculty].transform_keys do |key|
+        key == "archived" ? "archived_at" : key # map archived to archived_at
+      end
+    end
```

Comment:
This is not required; You can have `virtual` properties for the form and decide what it translates into during the save.